src/handlers/swap_handlers.ts

```diff
@@ -166,6 +177,15 @@ const parseGetSwapQuoteRequestParams = (req: express.Request): GetSwapQuoteReque
             ? undefined
             : parseStringArrForERC20BridgeSources(req.query.excludedSources.split(','));
     const affiliateAddress = req.query.affiliateAddress;
+    const rfqt =
+        req.query.intentOnFilling === undefined
+            ? undefined
+            : {
+                  intentOnFilling: ['true', ''].includes(req.query.intentOnFilling) ? true : false,
```

Comment:
Can we just do something like the following?

```
const rfqt = req.query.intentOnFilling === 'true'  ? { intentOnFilling: true } : undefined;
```

Reply:
I did some digging when I updated the schemas here.  I had initially wanted to make them `type: "boolean"` (whatever exactly that would mean!) but the `jsonschema` package didn't handle that nicely.  Later discovered this question which is a good analog to what we're doing here: https://github.com/OAI/OpenAPI-Specification/issues/1782

I changed this here because I liked the idea of the mere presence of this flag indicating it should be true, eg `/quote?intentOnFilling&...`.

Reply:
How we resolve https://github.com/0xProject/0x-api/pull/162#discussion_r409163018 will impact this